packages/docusaurus-theme-translations/package.json

```diff
@@ -23,9 +23,9 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.0.0-beta.9",
-    "chalk": "^4.1.2",
     "fs-extra": "^10.0.0",
-    "lodash": "^4.17.20"
+    "lodash": "^4.17.20",
+    "picocolors": "^1.0.0"
```

Comment:
does it need to be there? can't we use our logger package in this case?

Reply:
This one is because the theme-translations has a CLI that's only run during generation of translation messages. I'll move them to devDeps anyways